src/natcap/invest/urban_nature_access.py

```diff
@@ -125,7 +125,7 @@
                 "pop_[POP_GROUP]": {
                     "type": "ratio",
                     "required": (
-                        f"(search_radius_mode == '{RADIUS_OPT_POP_GROUP}') "
+                        f"(search_radius_mode.value == '{RADIUS_OPT_POP_GROUP}') "
```

Comment:
I suspect this won't have the intended effect if validation doesn't yet evaluate conditonal requirement in vector fields: https://github.com/natcap/invest/blob/9e50106f64bd0525163e347030c4d439bb4b910f/src/natcap/invest/validation.py#L847-L849

Reply:
I implemented conditional validation by rewriting the `required` attribute for vectors and CSVs to a boolean, which meant that this function doesn't have to change at all.  The rewriting happens within `validate()`, where we have easy access to the args themselves.  I also removed this comment about conditional requirement in cb70a12c7.